### PR TITLE
Arrays: added missing flatten, isList, normalize, pick, some, every, map

### PR DESCRIPTION
**Pull request by f3l1x**
This PR docs missing Arrays method.

### CODE REVIEW

**Comment by dg** [comment]
Can you change `.{toc: flatten()}` etc to `.[method]`?

**Comment by f3l1x** [comment]
Sure, sure. I've had this prepared before `.[method]` exist :-)

**Comment by f3l1x** [comment]
Related with:
- https://github.com/nette/docs/issues/213
- https://github.com/nette/docs/issues/275

File: cs/arrays.texy [hunk 1]
```diff
@@ -130,4 +130,101 @@ Arrays::renameKey($array, 'first', 'renamed');
...
+// $array = ['first' => null, 'a' => 'second'];
+\--
+
+/--php
+$array = Arrays::normalize([1 => 'first', 'a' => 'second'], 'foobar');
+// $array = ['first' => 'foobar', 'a' => 'second'];
+\--
+
+
+pick(array &$arr, string|integer $key, mixed $default = null): mixed .[method]
```
Comment (dg) [inline cs/arrays.texy]: We should use only short variant, i.e. `int`

File: cs/arrays.texy [hunk 2]
```diff
@@ -130,4 +130,101 @@ Arrays::renameKey($array, 'first', 'renamed');
...
+$a = Arrays::pick($array, null);
+// $a = 'bar'
+$b = Arrays::pick($array, 'not-exists', 'foobar');
+// $b = 'foobar'
+$c = Arrays::pick($array, 'not-exists');
+// throws Nette\InvalidArgumentException
+--\
+
+
+some(array $arr, callback $callback): bool .[method]
```
Comment (dg) [inline cs/arrays.texy]: it is `callable`

File: cs/arrays.texy (hunk 1 above)
Comment (f3l1x) [inline cs/arrays.texy]: Done

File: cs/arrays.texy (hunk 2 above)
Comment (f3l1x) [inline cs/arrays.texy]: Done

**Comment by dg** [comment]
Thanks

**Comment by f3l1x** [comment]
Houston we have a problem!

prace_s_poli_ _nette_utils_arrays___nette_framework

**Comment by dg** [comment]
fixed

**Pull request merged**